### DISCUSSION TITLE
Not able to install tailwind css for create-react-app

### DISCUSSION THREAD

**Discussion by kaffan** [tailwindlabs/tailwindcss#7236] [Help]
I am not able to install tailwind css for my react app
whenever I am running this command 'npm install -D tailwindcss postcss autoprefixer'
I am getting this message
added 14 packages, removed 1 package, and audited 1302 packages in 7s

172 packages are looking for funding
  run `npm fund` for details

8 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

**Comment by kncs** [comment]
Hello @kaffan, sounds like a normal post install message. Why do you think you are not able to install tailwind ?
